File: conditionals/time_of_day.py
# Chiara Wilson, Time of day project
import time
current=time.time()
local_time=time.localtime(current)
day = local_time.tm_wday
hour = local_time.tm_hour

if hour >= 0 and hour <= 11:
    print("Good Morning!\n")
elif hour == 12 and hour <= 16:
    print("Good afternoon!\n")
elif hour==17 and hour <=20:
    print("Good evening\n")
else:
    print("Good night\n")

File: conditionals/time_of_day.c
// Chiara Wilson, TEMPLATE
#include <stdio.h>
#include <time.h>
int num;

int main(void){
time_t rawtime;
struct tm * timeinfo;

   time_t now= time(NULL);

struct tm* tm_struct = localtime(&now);
int hour= tm_struct->tm_hour;

if(hour>=0 && hour<=11){
    printf("Good morning!\n");
}else if( hour== 12 && hour <=16){
    printf("Good afternoon!\n");
}else if ( hour ==17 && hour <= 20){
    printf("Good evening! \n");
}else{
    printf("Good night\n");
}
    return 0; 
} 